render.yaml

```diff
@@ -0,0 +1,225 @@
+# Configure deployment on Render.com infrastructure.
+# For info on options, see: https://render.com/docs/blueprint-spec
+#
+# THIS IS AN EXPERIMENT. Our primary deployment is still on AWS, but we hope to
+# switch to this if it works well and is not too expensive. See:
+# https://github.com/usdigitalresponse/univaf/issues/683
+
+databases:
+  - name: univaf-db-production
+    plan: pro
+    databaseName: univaf_db_production
+    user: univaf
+    # Not sepecifying IPs for now; we may want to set this to an empty array
+    # to disallow public connections, though. i.e. `ipAllowList: []`
+    # ipAllowList:
+    #   - source: 203.0.113.4/30
+    #     description: office
+
+services:
+  - name: API Server
+    type: web
+    plan: standard
+    env: node
+    repo: "https://github.com/usdigitalresponse/univaf.git"
+    scaling:
+      minInstances: 1
+      maxInstances: 6
+      targetMemoryPercent: 60 # optional if targetCPUPercent is set
+      targetCPUPercent: 60 # optional if targetMemory is set
```

Comment:
`standard` is 1 CPU and 2GB RAM, which is the same as we have the API service currently configured to use in AWS.

This is also a pretty similar scaling configuration to what we currently have: https://github.com/usdigitalresponse/univaf/blob/fd6076ad7e71046a9097105979221c281fc93324/terraform/auto_scaling.tf#L3-L89

The biggest difference is that, on AWS, we set exact thresholds. On Render, we set an optimal usaage level and it decides where the thresholds are: https://render.com/docs/scaling